web/src/components/Error.tsx

```diff
@@ -56,6 +56,10 @@ const ErrorComponent: React.FC<Props> = (props) => {
 
     return (
       <ListGroup variant="flush">
+        <ListGroup.Item>
```

Comment:
for consistency, shall we render the ListGroup.Item inside `renderTimeOccurrence `?